mmdet/models/roi_heads/test_mixins.py

```diff
@@ -114,8 +114,8 @@ def simple_test_bboxes(self,
         cls_score = cls_score.reshape(batch_size, num_proposals_per_img, -1)
 
         if not torch.onnx.is_in_onnx_export():
-            # remove padding
-            supplement_mask = rois[..., -1] == 0
+            # remove padding, ignore batch_index when calculating mask
+            supplement_mask = rois.abs()[..., 1:].sum(dim=-1) == 0
```

Comment:
Thanks for the contribution. why need `1:`

Reply:
the zero dimension is batch idx